api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -76,6 +77,12 @@
       return formatters;
    }
 
+   public static Charset getDefaultEncoding()
```

Comment:
It's a default implementation of original code in the JavaParserImpl.java. So I didn't change it.